files/www/cgi-bin/sysinfo.json

```diff
@@ -244,6 +244,33 @@ if string.find(nixio.getenv("QUERY_STRING"):lower(),"services=1") then
 	info["services"]=services
 end
 
+-- LOCALLY HOSTED SERVICES ONLY
+if string.find(nixio.getenv("QUERY_STRING"):lower(),"services_local=1") then
```

Comment:
I would prefer "local_services" instead.

Reply:
I could just swap the info[] line to read services_local instead. :smile: